crates/adapters/src/server/mod.rs

```diff
@@ -749,6 +908,72 @@ outputs:
         drop(resp1);
         drop(resp2);
 
+        // Request quantiles.
+        let mut quantiles_resp1 = server
+            .get("/egress/test_output1?mode=snapshot&query=quantiles")
+            .send()
+            .await
+            .unwrap();
+        assert!(quantiles_resp1.status().is_success());
+        let body = quantiles_resp1.body().await;
+        // println!("Response: {body:?}");
+        let body = serde_json::from_slice::<JsonValue>(&body.unwrap()).unwrap();
+        println!("Quantiles: {body}");
+
+        // Request quantiles for the input collection -- inputs must also behave as outputs.
```

Comment:
this could also be achieved by the user defining views for all inputs they want to monitor.